### PR TITLE
DEVO-253 Deprecation warning fixes

### PR DESCRIPTION
**Pull request by sensei100**
- Refactors rails and app dependencies tasks to remove loop

### CODE REVIEW

File: tasks/dependencies.yml [hunk 1]
```diff
@@ -88,7 +85,6 @@
 
 - name: Install app specifc dependencies
   yum:
-    name: "{{ item }}"
+    name: "{{ rails_app_additional_dependencies | default([]) }}"
```
Comment (dkinzer) [inline tasks/dependencies.yml:88]: I'm worried about this one so will wait to get @bibliotechy's input.  Wouldn't this be a string "[]" vs an actual array []?

Comment (sensei100) [reply tasks/dependencies.yml:88]: @dkinzer I think that is how it is supposed to be, I just moved what we already had on line 93.  It is written with the quotes there too.

Comment (sensei100) [reply tasks/dependencies.yml:88]: But we can wait for Chad if you want.

Comment (dkinzer) [reply tasks/dependencies.yml:88]: Hmm.. OK, I guess if everything is treated as text.

**Pull request merged**